app/assets/javascripts/hydra-editor/keyboard_support.js

```diff
@@ -1,14 +0,0 @@
-Blacklight.onLoad(function() {
-  $('body').on('keypress', '.multi-text-field', function(event) {
-    var $activeField = $(event.target).parents('.field-wrapper'),
-        $activeFieldControls = $activeField.children('.field-controls'),
-        $addControl = $activeFieldControls.children('.add'),
-        $removeControl = $activeFieldControls.children('.remove');
-    if (event.keyCode == 13) {
-      event.preventDefault();
-      $addControl.click()
-      $removeControl.click()
-    }
-  });
-});
-
```

Comment:
Yeah our forms need a good looking into for usability/accessibility. So this code removes the default ability to add and remove metadata by hitting the enter key and gives it only to the button that submits the form. Does this mean that I have to use my mouse throughout the form to add and edit metadata or select items from a dropdown? And then get to use the enter key only to submit the form? If I'm following along, I think that there might be a better way to approach submitting the form using only a keyboard.